vm/vm.go

```diff
@@ -544,8 +554,22 @@ func (vm *VM) startNormalOp(ctx context.Context) error {
 		return fmt.Errorf("failed to add tx gossip handler: %w", err)
 	}
 	vm.checkActivity(ctx)
-	vm.normalOp.Store(true)
 
+	lastAccepted, err := vm.LastAcceptedBlock(ctx)
+	if err != nil {
+		return err
+	}
+	executionBlk, err := vm.GetExecutionBlock(ctx, lastAccepted.GetID())
+	if err != nil {
+		return err
+	}
+
+	isValidityWindowComplete := vm.chainTimeValidityWindow.Complete(ctx, executionBlk)
+	if !isValidityWindowComplete {
+		return errors.New("critical error: validity window's partial state may lead to inconsistencies")
```

Comment:
```suggestion
		return errors.New("cannot start normal operation with incomplete validity window")
```

can we declare this error at the top of the file in a var block to match the existing style?

Reply:
Done: https://github.com/ava-labs/hypersdk/blob/94538b1c8919e9bde96ef52ae888b3c42d0d22c2/vm/vm.go#L66

https://github.com/ava-labs/hypersdk/blob/94538b1c8919e9bde96ef52ae888b3c42d0d22c2/vm/vm.go#L576